### PR TITLE
iOS fixes

### PR DESCRIPTION
**Pull request by thomassth**
#302 but in a separate branch. See if this is better.

### CODE REVIEW

**Comment by nickrandolph** [comment]
I think since we merged the timezone pr that there's now conflicts on this PR that are preventing a merge:
Conflicting files
example/ios/.symlinks/plugins/device_calendar
example/lib/presentation/pages/calendars.dart

File: example/ios/.symlinks/plugins/device_calendar [hunk 1]
```diff
@@ -1 +1 @@
-/Volumes/MacData/Codebase/Flutter/device_calendar/
```
Comment (nickrandolph) [inline example/ios/.symlinks/plugins/device_calendar:1]: @thomassth is this change in folder name required?

**Comment by andzejsw** [comment]
In my opinion no generated files should be commited and should be added to ".gitignore". Since "ios/.symlinks" folder is generated, it with all of its contend should be gitignored. Do you agree guys?

**Comment by thomassth** [comment]
i've added it into .gitignore and caught up with upstream.

**Comment by nickrandolph** [comment]
@thomassth these changes look fine but can we link to the relevant issue here - I'm not sure what exactly we're fixing

**Comment by thomassth** [comment]
I was looking at [this comment](https://github.com/flutter/flutter/issues/16049#issuecomment-611192738).

The whole thread was repeatedly referenced in out issues and readme.

Not sure what it fixes, as I didn't see any error on my side before, but it seems needed.

**Comment by nickrandolph** [comment]
@andzejsw  and @thomassth are you ok with this PR?

**Comment by andzejsw** [comment]
All looks good to me

**Review by sowens-csd — APPROVED** [review]
Changes look safe to me. Ran successfully on both an iOS 15 simulator and a real device.

**Pull request merged**